store.go

```diff
@@ -2750,7 +2798,7 @@ func (s *store) LayerParentOwners(id string) ([]int, []int, error) {
 }
 
 func (s *store) ContainerParentOwners(id string) ([]int, []int, error) {
-	rlstore, err := s.LayerStore()
+	rlstore, err := s.RWLayerStore()
```

Comment:
This could be RO as we're not changing state.

Reply:
`ParentOwners` is currently not part of the `ROLayerStore` interface because we would require the mount information for that method.

Reply:
FWIW, the `...ParentOwners()` family of methods was an experiment that we ended up not using, so if/when we want to break API, we should junk them.

Reply:
I can follow-up on that in another PR :)